packages/documentation-site/patternfly-docs/content/PatternFly-AI/conversation-design/conversation-design.md

```diff
@@ -0,0 +1,75 @@
+---
+id: Conversation design
+section: PatternFly-AI
+---
+
+import { Button, Flex, FlexItem } from '@patternfly/react-core';
+import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
+
+
+# Conversation design guidelines
+
+**Conversation design** is method of writing for conversational interfaces, like chatbots or voicebots. The goal of conversation design is to create an interactive experience that resembles human-to-human conversation as much as possible. Like traditional content design, conversation design is focused on using words to make experiences clear, concise, and well-timed.
+
+Good conversation design combines content strategy, writing, and design to ensure that the development and use of chatbots is centered on real user needs. This includes making sure that all text related to the conversational UI supports easy engagement with chatbots, while staying aligned with brand standards.
+
+When designing AI conversations alongside PatternFly-based projects, it is important to align with our established [brand voice and tone](/ux-writing/brand-voice-and-tone), as well as our [ethical guidelines for AI.](/patternfly-ai/ai-guidelines)
+
+## Best practices 
+
+Following these best practices to help ensure that your users can complete their goals through an AI-based conversation: 
+
+- Be transparent about AI use. 
+- Be direct, brief, and consistent. 
+- Frame questions in terms of value for the user.
+- If you ask for personal info, tell users "why" you're asking first. 
+- Always have the last word.
+
+## Writing for chatbots
+
+When chatbots are design to meet the needs of your users, they can improve the overall UX of your product, because they are convenient, efficient, and persistent. 
+
+[Our virtual assistant extension](/patterfnfly-ai/chatbot/about-virtual-assistant) utilizes PatternFly components to create a foundation for an AI-based chatbot, with additional customization options.
```

Comment:
```suggestion
[Our virtual assistant extension](/patternfly-ai/chatbot/about-virtual-assistant) utilizes PatternFly components to create a foundation for an AI-based chatbot, with additional customization options.
```

Reply:
This link doesn't exist either though; I think we are using patternfly-ai/chatbot/about-chatbot?